mrp_unbuild_cust_alu_analytics/models/mrp_unbuild.py

```diff
@@ -0,0 +1,151 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3.0 (https://www.gnu.org/licenses/agpl-3.0.html)
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+import os
+from lxml import etree
+import logging
+
+class MrpUnbuild(models.Model):
+    _inherit = "mrp.unbuild"
+
+    process_type_id = fields.Many2one('mrp.unbuild.process.type', string='Process type')
+    incidence_ids = fields.One2many(
+        comodel_name="mrp.unbuild.incidence",
+        inverse_name="unbuild_id",
+        string="Incidences",
+    )
+    analytic_ids = fields.One2many(
+        comodel_name="qc.inspection",
+        string="Analytics",
+        store=False
```

Comment:
From what I have read, if any inverse_name is provided; it will link anyway but  the Many2one inverse field will not be created.
But yes, the correct is as u said, mostly to be bidirectional.

Reply:
Fixed